EvaluateBracket.java

```diff
@@ -0,0 +1,70 @@
+import java.util.Stack;
+
+public class EvaluateBracket {
+
+	public EvaluateBracket() {
+		// TODO Auto-generated constructor stub
+	}
+	
+	public static boolean evaluate(String inputString){
+		Stack<String> bracketStack = new Stack<String>();
+		if(inputString == null)
+			return false;
+		if(inputString == "")
+			return false;
+		for(int i=0; i< inputString.length();i++){
+			
+			String s = Character.toString(inputString.charAt(i));
+			if("{".equals(s) || "[".equals(s)|| "(".equals(s))
+				
+			bracketStack.push(s);
+			
+			else{ 
+				  if(bracketStack.isEmpty())
+					  return false;
+					if( "}".equals(s) ){				
+						if ( !"{" .equals(bracketStack.pop()))
+								
+							return false;
+						
+					}
+					else if ("]".equals(s)){
+						if (! "[" .equals(bracketStack.pop()))
+							return false;
+					}
+					else if (")".equals(s)){
+						if ( ! "(" .equals(bracketStack.pop()))
+							return false;
+					}
+								 						
+				}
+		}
+		if(!bracketStack.isEmpty())
```

Comment:
can we change line 42-44 to just one line?? this is called reducing boilerplate code, we need to try writing less lines of code and simplify wherever its possible.
return bracketStack.isEmpty()